linuxNew/jdk/suse/src/main/packaging/temurin/8/temurin-8-jdk.spec

```diff
@@ -220,5 +229,5 @@ fi
 %{prefix}
 
 %changelog
-* Tue Aug 31 2021 Eclipse Adoptium Package Maintainers <temurin-dev@eclipse.org> 8.0.302.0.0.8-1.adopt0
+* Fri Aug 31 2021 Eclipse Adoptium Package Maintainers <temurin-dev@eclipse.org> 8.0.302.0.0.8-1.adopt0
```

Comment:
Aug 31 was actually a `Tue`. Produces a `bogus date in changelog` warning.

Reply:
Updated. Though this log will need to be updated to 8u312-b07 version. The PR https://github.com/adoptium/installer/pull/357/files didn't update changelog. Will do that in a separate PR.